README.md

```diff
@@ -120,13 +241,15 @@ Changes are exported in batches of a size defined by the `batch-size` flag. By d
 
 This command has two modes: bounded and unbounded.
 
-##### Bounded
+#### **Bounded**
  If both a start and end ledger are provided, then the command runs in a bounded mode. This means that once all the ledgers in the range are processed and exported, the command shuts down.
  
-##### Unbounded
+#### **Unbounded**
 If only a start ledger is provided, then the command runs in an unbounded fashion starting from the provided ledger. In this mode, the Stellar Core connects to the Stellar network and processes new changes as they occur on the network. Since the changes are continually exported in batches, this process can be continually run in the background in order to avoid the overhead of closing and starting new Stellar Core instances.
 
-#### export_orderbooks
+<br>
+
+### **export_orderbooks**
```

Comment:
Drop all references to orderbooks. This was an experimental feature that we've never used.

Reply:
Or mention that it's experimental and unsupported

Reply:
Marked it as unsupported